src/main.js

```diff
@@ -11,7 +11,7 @@ if (require.main === module) {
 }
 
 export function createInstance(config=cli.argv) {
-  log.level(config.logLevel);
+  log.level = config.logLevel;
```

Comment:
Okay, it [defines a default](https://github.com/mozilla/dispensary/blob/b05dfcac71e22a3061ea0ab3f08d66f527801229/src/cli.js#L16) so we should be good.